packages/frontend-plugin-api/report.api.md

```diff
@@ -922,6 +870,13 @@ export { ErrorApiErrorContext };
 
 export { errorApiRef };
 
+// @public (undocumented)
+export const ErrorDisplay: ((
+  props: CoreErrorDisplayProps,
+) => JSX.Element | null) & {
```

Comment:
Did you decide against having a `SwappableComponentType` type or so? Which basically encapsulates this slight mess.